core/trino-main/src/test/java/io/trino/sql/analyzer/TestAnalyzer.java

```diff
@@ -990,7 +1183,7 @@ public void testWindowFrameTypeRange()
 
         assertFails("SELECT array_agg(x) OVER (RANGE BETWEEN 1 PRECEDING AND 1 FOLLOWING) FROM (VALUES 1) T(x)")
                 .hasErrorCode(MISSING_ORDER_BY)
-                .hasMessage("line 1:21: Window frame of type RANGE PRECEDING or FOLLOWING requires ORDER BY");
+                .hasMessage("line 1:27: Window frame of type RANGE PRECEDING or FOLLOWING requires ORDER BY");
```

Comment:
Why did this change?

Reply:
The error points to the beginning of window specification instead of the word `OVER` because not every window specification is now preceded with `OVER`.